customisation/glossaries.md

```diff
@@ -5,3 +5,45 @@ layout: coming_soon
 title: Glossaries
 description: Terminology lists for training and customising machine translation
 ---
+#Glossaries
+
+Glossaries are lists of terms in one language aligned to their translation in one or more languages. They can be added to machine translation systems.
+Adding a glossary to a commercial machine translation provider can increase the terminology accuracy of the output.
+
+## Application
+
+Glossaries are leveraged at translation time, and no training is required.
+When a glossary term is matched in the source sentence, its translation is enforced in the output.
+
+The insertion of a glossary term in the output happens regardless of the context.
+The terms included in the glossary should therefore be unambiguous and context-independent, as in the following examples:
+- Respiratory syndrome
+- Balance sheet
+- Paraffin
+
+Ambiguous terms can have a negative impact on the output:
+- Spread
+- Cookies
+- Surface treatment
+
+##Term matching and insertion
+
+Several machine translation providers offer the possibility to enhance their output using a glossary.
+The way in which the glossary is leveraged may differ in terms of source term matching and target term insertion.
+
+In some cases the source term is matched in the source text only if it occurs with the same morphology and ortography as in the glossary.
```

Comment:
What do you think about something like this?

> Approaches to term matching:
> - Strict exact term matching: ...
> - Morphology-aware term matching: ...
> - 

Or 

> Strict exact term matching is a problem because terms can have varying forms.
> 
> With some machine translation systems, term matching can handle common variants.
> - Morphology
> - Orthography

By orthography here do you mean like American/British spelling differences, or more?  We could replace orthography with the actual issues.

> - Spelling variants
> - Uppercase and lowercase